packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -479,6 +491,7 @@ export class DocumentsSchemaController {
 		this.desiredSchema = {
 			version: currentDocumentVersionSchema,
 			refSeq: documentMetadataSchema?.refSeq ?? 0,
+			minVersionForCollab: documentMetadataSchema?.minVersionForCollab ?? minVersionForCollab,
```

Comment:
Should this take the max of the two if both present? To advance the min over time like we've been discussing